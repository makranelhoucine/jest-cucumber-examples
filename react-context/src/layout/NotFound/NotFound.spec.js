import * as React from "react";
import * as ReactDOM from "react-dom";
import * as renderer from "react-test-renderer";

import Component from "./";

describe("Layout: Footer should...", () => {
  test("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<Component />, div);
  });

  test("Render match snapshot", () => {
    const component = renderer.create(<Component />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
