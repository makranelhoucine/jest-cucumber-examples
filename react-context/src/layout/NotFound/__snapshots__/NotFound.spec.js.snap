// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout: Footer should... Render match snapshot 1`] = `
<h1>
  Nous n'avons pas trouv√© votre page.
</h1>
`;
